core/arch/arm/kernel/generic_boot.c

```diff
@@ -55,6 +56,11 @@
 #include <kernel/vfp.h>
 #endif
 
+#if defined(CFG_DT)
+#include <libfdt.h>
+#define DTB_MAX_SIZE		0x10000
```

Comment:
That might be platform-dependent I suppose, so I suggest making it a config value too -- i.e., just set `CFG_DTB_MAX_SIZE ?= 0x10000` in `mk/conf.mk`.